src/Symfony/Component/Uid/Command/GenerateUlidCommand.php

```diff
@@ -25,12 +25,9 @@
 #[AsCommand(name: 'ulid:generate', description: 'Generate a ULID')]
 class GenerateUlidCommand extends Command
 {
-    private UlidFactory $factory;
-
-    public function __construct(UlidFactory $factory = null)
-    {
-        $this->factory = $factory ?? new UlidFactory();
-
+    public function __construct(
+        private UlidFactory $factory = new UlidFactory(),
```

Comment:
```suggestion
        private readonly UlidFactory $factory = new UlidFactory(),
```
Can we also add readonly where we can?

Reply:
I think @nicolas-grekas is 👎 on this

Reply:
It'd be useless or just visual debt in practice to me yes.